packages/subiquity_client/lib/src/types.freezed.dart

```diff
@@ -12,7 +12,7 @@ part of 'types.dart';
 T _$identity<T>(T value) => value;
 
 final _privateConstructorUsedError = UnsupportedError(
-    'It seems like you constructed your class using `MyClass._()`. This constructor is only meant to be used by freezed and you are not supposed to need it nor use it.\nPlease check the documentation here for more information: https://github.com/rrousselGit/freezed#adding-getters-and-methods-to-our-models');
+    'It seems like you constructed your class using `MyClass._()`. This constructor is only meant to be used by freezed and you are not supposed to need it nor use it.\nPlease check the documentation here for more information: https://github.com/rrousselGit/freezed#custom-getters-and-methods');
```

Comment:
This seems to have made it in by accident, I guess?

Reply:
Yeah, had to run `pub upgrade`, should be fixed now.

Reply:
```
[ubuntu_bootstrap]: You have hit a bug in build_runner
```
:upside_down_face: